docs/setting-up-a-validator.md

```diff
@@ -149,6 +149,29 @@ ExecStart=/usr/bin/gbt orchestrator \
 
 For the Geth node, if you are going to run a geth full node delete lines 11-15 and uncomment lines 17-21
 
+## Add Skip Invariants to Gravity Command
+This will be fixed in a future upgrade, if it is not already fixed follow these steps. </br> 
+Look at the service run script for gravity-node, which should be located at `/etc/systemd/system/gravity-node.service` </br> 
+Update the `ExecStart` line to look like the following: 
+
+```
+ExecStart=/usr/bin/gravity start --x-crisis-skip-assert-invariants
+```
+
+## Update Geth Service ExecStart Reference
+In the previous step we moved geth to /usr/sbin but the ExecStart may stil reference it in /usr/bin/ </br> 
+Look at the reference by opening the following file: </br>
+```
+/etc/systemd/system/geth.service
```

Comment:
this is a good catch! but shouldn't we update the service file to match rather than instruct the user to go fix it?

Reply:
Okay yepp,
Going to remove this doc instruction and add the amendment to https://github.com/Gravity-Bridge/Gravity-Docs/blob/main/configs/geth.service in this PR

Reply:
Changed to /sbin configs/geth.service

Reply:
good I think that means this whole section should go. Someone who already has the wrong path will not be re-reading this segment.